python.d/go_expvar.chart.py

```diff
@@ -0,0 +1,225 @@
+# -*- coding: utf-8 -*-
+# Description: go_expvar netdata python.d module
+# Author: Jan Kral (kralewitz)
+
+from __future__ import division
+from base import UrlService
+import json
+
+# default module values (can be overridden per job in `config`)
+# update_every = 2
+priority = 60000
+retries = 60
+
+
+MEMSTATS_CHARTS = {
+    'memstats.heap': {
+        'options': ['heap', 'memory: size of heap memory structures', 'kB', 'memstats', 'expvar.memstats.heap', 'line'],
+        'lines': [
+            ['memstats.heap_alloc', 'alloc', 'absolute', 1, 1024],
+            ['memstats.heap_inuse', 'inuse', 'absolute', 1, 1024]
```

Comment:
Changed the wiki and the config file sample in [718cda1c](https://github.com/firehol/netdata/pull/2280/commits/718cda1c70f23cb11b6fe06e5b07f70e1a485d74)